controllers/articles.controllers.js

```diff
@@ -17,3 +20,16 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const article_id = req.params.article_id;
+  const promises = [
+    selectCommentsByArticleId(article_id),
+    selectArticleById(article_id),
```

Comment:
nice use of your `selectArticleById` model function and very clean with a `Promise.all` well done!